app/models/user_mentee_application.rb

```diff
@@ -40,7 +40,7 @@ class UserMenteeApplication < ApplicationRecord
                           }, class_name: 'MenteeApplicationState', dependent: nil
   # rubocop:enable Rails/InverseOf
 
-  validates :available_hours_per_week, numericality: { greater_than: 0, less_than_or_equal_to: 60 }
+  validates :available_hours_per_week, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 60 }
```

Comment:
So the idea here was to have it still validate at `greater_than: 0` and force them to consciously put in a number without just using the default. I'm kind of not opposed though to just allowing 0 lol.